opnode/l2/source.go

```diff
@@ -246,3 +248,61 @@ func PayloadToBlockRef(payload *ExecutionPayload, genesis *rollup.Genesis) (eth.
 		L1Origin:   l1Origin,
 	}, nil
 }
+
+type ReadOnlySource struct {
+	rpc     *rpc.Client       // raw RPC client. Used for methods that do not already have bindings
+	client  *ethclient.Client // go-ethereum's wrapper around the rpc client for the eth namespace
+	genesis *rollup.Genesis
+	log     log.Logger
+}
+
+func NewReadOnlySource(ll2Node *rpc.Client, genesis *rollup.Genesis, log log.Logger) (*ReadOnlySource, error) {
```

Comment:
is `ll2` (two L) intentional? Typo? (might be my own typo)

Reply:
looks like it's a typo in `Source` as well, fixed.